src/components/Map/__tests__/MapWithGoogleLoader.spec.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { MapWithGoogleLoader } from '..';
+import '@testing-library/jest-dom';
+
+describe('<MapWithGoogleLoader/> that loads google api and renders a Map', () => {
+    it('should render with default props', () => {
+        const view = render(<MapWithGoogleLoader apiKey="someKey" />);
+
+        expect(view.container).toMatchSnapshot();
+        expect(screen.getByRole('status')).toBeInTheDocument();
```

Comment:
The snapshot shows a loader. Maybe that is expected, then we should adjust the title of this test. 
If we can render something more specific to the map, that would be great to test. However, since we don't actually want to call google from unit test, but stab the object, I'm not sure what tests we can do here at all. Any ideas?

Reply:
In fact, I believe we don't require tests specifically for `LoadScriptNext` since it serves as a wrapper for third-party component and `testing-library.com` guideline principle says "you should avoid mocking out component". Thus, I'd like to remove that test if you don't mind.

Reply:
Hmm, it seems that even the `react-google-maps/api` library lacks comprehensive tests. :) https://github.com/JustFly1984/react-google-maps-api/tree/develop/packages/react-google-maps-api/src

Reply:
yes, that makes sense to me too